tests/legacy-cli/e2e/utils/registry.ts

```diff
@@ -54,3 +57,19 @@ export function createNpmConfigForAuthentication(
       `,
   );
 }
+
+export function setNpmEnvVarsForAuthentication(
+  /** When true, an incorrect token is used. Use this to validate authentication failures. */
+  invalidToken = false,
+): void {
+  const token = invalidToken ? `invalid=` : validToken;
+  const registry = `http://localhost:4876/`;
```

Comment:
Can you extract into a shared const, since it's being used in multiple places
Something along `SECURE_REGISTRY`.

Reply:
Definitely.